OmniKitUI/ViewControllers/CommandResponseViewController.swift

```diff
@@ -27,15 +27,11 @@ extension CommandResponseViewController {
                     }).joined(separator: "\n")
 
                     if messageWithRecovery.isEmpty {
-                        if error.localizedDescription.isEmpty {
-                            response = String(describing: error)
-                        } else {
-                            response = error.localizedDescription
-                        }
+                        response = error.localizedDescription
                     } else {
                         response = messageWithRecovery
                     }
-                } else if let localizedError = error as? LocalizedError, !localizedError.localizedDescription.isEmpty {
+                } else if let localizedError = error as? LocalizedError {
```

Comment:
I think the whole conditional can be simplified:
```
} else if let localizedError = error as? LocalizedError {
    response = localizedError.localizedDescription
} else if let error = error {
    response = String(describing: error)
}
```
Can be turned into:
```
} else if let error = error {
    response = error.localizedDescription
}
```

Reply:
Makes sense if localizedDescription.isEmpty is always false

Reply:
I should've mentioned there are several of these in this PR that could be simplified.

Reply:
Done. Now that PodCommsError.commsError.localizedDescription has a more meaningful content, the simplified version will display a meaningful error message for RileyLink errors with this more simplified method.